src/scraper.rs

```diff
@@ -123,10 +128,10 @@ impl Scraper {
             response::ResponseData::Other(data) => data,
         };
 
-        //Create a scope to unlock path_map automagicly
```

Comment:
Did you remove the "lock-scope" ?

Reply:
Do we use `path_map` after this point anywhere within `handle_url`? If not, then it's not needed as it will be unlocked anyways at: https://github.com/Skallwar/suckit/blob/ae20a2713e83c91983a2d9bbbec0e94092faee89/src/scraper.rs#L149 Correct if I am wrong

Reply:
> Do we use `path_map` after this point anywhere within `handle_url`?

Other thread are maybe waiting for it. The sooner it is released the better it is

Reply:
Reverted back